mycore-base/src/main/java/org/mycore/common/content/streams/MCRByteArrayOutputStream.java

```diff
@@ -38,6 +38,7 @@ public MCRByteArrayOutputStream(int i) {
     }
 
     public byte[] getBuffer() {
-        return super.buf;
+        return buf.clone();
     }
```

Comment:
This method is generally confusing. `ByteArrayOutputStream` itself offers the method `#toByteArray`. This is also synchronized and should generally be used.

Reply:
True. Not sure why we even have this class tbh.
@yagee-de should this class be removed?